CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+4.0.0 - 2019-06-28
```

Comment:
I think this should be 3.1.0, which is a little counter intuitive, but since we're not breaking anything in the existing library there's no need for a major version bump...